modules/custom/az_paragraphs/az_paragraphs.module

```diff
@@ -22,3 +22,16 @@ function az_paragraphs_help($route_name, RouteMatchInterface $route_match) {
     default:
   }
 }
+
+/**
+ * Implements hook_theme_registry_alter().
+ */
+function az_paragraphs_theme_registry_alter(&$theme_registry) {
+  if ($theme_registry['field_multiple_value_form']) {
+    if ($theme_registry['field_multiple_value_form']['preprocess functions']) {
```

Comment:
Nitpick:  I think it might be preferable to use `isset()` here:
```suggestion
    if (isset($theme_registry['field_multiple_value_form']['preprocess functions'])) {
```